### PR TITLE
Check redirect url

### PR DESCRIPTION
**Pull request by mingwli**
#3943 
config.redirections dest must start with '/'

### CODE REVIEW

File: docs/specs/resolve.yml [hunk 1]
```diff
@@ -168,17 +168,6 @@ outputs:
   docs/a.json: |
     {"content":"<p><a href=\"/absolute/path?a#b\">redirect</a></p>"}
 ---
-# Files redirected to a relative URL
```
Comment (OsmondJiang) [inline docs/specs/resolve.yml]: this case should be added by reason?

Comment (OsmondJiang) [reply docs/specs/resolve.yml]: impact test will test us it's used or not :)

Comment (mingwli) [reply docs/specs/resolve.yml]: there does exist urls other than `docs.microsoft.com`.
```text
articles/machine-learning/service/aml-pricing.md: 'https://azure.microsoft.com/pricing/details/machine-learning-services'
articles/machine-learning/studio/live-chat.md: 'https://social.msdn.microsoft.com/Forums/en-US/home?forum=MachineLearning'
```
how should we handle this?

Comment (yufeih) [reply docs/specs/resolve.yml]: these are simple redirections (without affecting document id)

**Pull request merged**